Cargo.toml

```diff
@@ -2,6 +2,3 @@
 members = [
     "ata"
 ]
-
-[profile.release.package.ata]
-strip = true
```

Comment:
Why was this removed? This it cause errors on Android?

Reply:
I'm actually not sure. It may be a bug in cargo add, it's also possible that I messed something up when editing these files in vim on Android 😂😂